backend/base/templates/org_locationgames.html

```diff
@@ -252,6 +263,22 @@
                     }, 2500); // Hide after 5000ms or 5 seconds
                 }
             });
+            $(document).ready(function () {
+                $('#nextTabBtn').click(function () {
+                    const activeTab = $('.tab-button.active');
+                    const nextTab = activeTab.next('.tab-button');
+                    
+                    if (nextTab.length > 0) {
+                        activeTab.removeClass('active');
+                        nextTab.addClass('active');
+                        
+                        const url = nextTab.data('url');
+                        $('#tab-content').load(url, function () {
+                            initFormSubmission();
+                        });
+                    }
+                });
+            });
```

Comment:
This block is getting repeated in most of your HTML, can you move this to common js functions?

Reply:
removed repeated lines of code